tests/test_default_browser.py

```diff
@@ -90,17 +88,13 @@ def test_default_safari(become_mac, change_linux_default):  # noqa: F811
 
 
 @pytest.mark.browser_name("chromehtml")
-def test_default_windows_chrome(
-    become_windows, change_win_default  # noqa: F811
-):
+def test_default_windows_chrome(become_windows, change_win_default):  # noqa: F811
     """Test that chrome is identified correctly on Windows"""
     assert utils.default_browser() == browsers.Chrome
 
 
-@pytest.mark.browser_name("firefoxurl")
-def test_default_windows_firefox(
-    become_windows, change_win_default  # noqa: F811
-):
+@pytest.mark.browser_name("firefoxurl-3EEDF34567DDE")
```

Comment:
Changing `firefoxurl` to `firefoxurl-3EEDF34567DDE` resulted in several lines that were being missed in `utils.py` to be covered.

Reply:
@Samyak2 I've fixed all the tests issues, hope you were not in process of doing so.

Reply:
> Changing `firefoxurl` to `firefoxurl-3EEDF34567DDE` resulted in several lines that were being missed in `utils.py` to be covered.

Oh F, how did that change? xD
I only did formatting there

> @Samyak2 I've fixed all the tests issues, hope you were not in process of doing so.

I hadn't started working on fixing it yet, so that's great. Thanks!